src/Esprima/Loc.cs

```diff
@@ -5,11 +5,11 @@ namespace Esprima;
 [DebuggerDisplay("{ToString()}")]
 public readonly struct Location : IEquatable<Location>
 {
-    public Position Start { get; }
-    public Position End { get; }
-    public string? Source { get; }
+    public readonly Position Start;
```

Comment:
again might be ref-readonly candidates, probably could be benchmarked separately like the previous if makes any difference

Reply:
I changed this to make it consistent with `Range`. Ref readonly with aggressive inlining would probably be just as performant, however, I think public fields just do fine in this case so why complicate it.